libs/sdk-core/src/breez_services.rs

```diff
@@ -1820,11 +1820,9 @@ impl BreezServices {
             .target(env_logger::Target::Pipe(target_log_file))
             .parse_filters(
                 r#"
-                debug,
-                breez_sdk_core::input_parser=warn,
-                breez_sdk_core::backup=info,
-                breez_sdk_core::persist::reverseswap=info,
-                breez_sdk_core::reverseswap=info,
+                info,
+                breez_sdk_core=debug,
+                sdk_common=debug,
```

Comment:
NIT: now that the root logger is set to `info`, do you think adding
```
breez_sdk_core::greenlight::node_api=debug,
```
would make sense?

Reply:
I thought that would be caught by `breez_sdk_core=debug`, but I'll check

Reply:
Checked and `breez_sdk_core::greenlight::node_api` logs are included this way